api/fileProcessing/index.ts

```diff
@@ -99,8 +99,7 @@ export default function (app: Application) {
     }
 
     if (success) {
-      const jobs = models.Recording.processingStates[recording.type];
-      const nextJob = jobs[jobs.indexOf(recording.processingState) + 1];
+      const nextJob = recording.getNextState();
```

Comment:
Does this really work for reprocess?  Jobs.indexOf("reprocess") will return -1 won't it?

Reply:
sorry i dont follow